docs/writing/bwa-mem_v4.xml

```diff
@@ -38,7 +38,7 @@
                  <param name="fastq_input1" type="data" format="fastqsanger" label="Select fastq dataset" help="Specify dataset with single reads"/>
             </when>
             <when value="paired_collection">
-                 <param name="fastq_input" format="fastqsanger" type="data_collection" collection_type="paired" label="Select dataset pair" help="Specify paired dataset collection containing paired reads" />
+                 <param name="fastq_input" format="fastqsanger" type="data_collection" collection_type="paired" label="Select a paired collection" label="Select dataset pair" help="Specify paired dataset collection containing paired reads"/>
```

Comment:
Looks like I screwed up the rebase; shouldn't have tried this will half listening to a talk.